react/features/conference/components/Conference.web.js

```diff
@@ -117,6 +122,11 @@ class Conference extends Component<*> {
 
                 <DialogContainer />
                 <NotificationsContainer />
+
+                { this.props._callOverlayVisible
+                    && <CallOverlay />
+                }
+
```

Comment:
I think I've mostly subscribed to the idea that `CallOverlay` is not an overlay in the terms of `OverlayContainer` (after you explained it to me in a call - thank you very much!).

I don't like though:

1. If it's not an overlay, then why is it still called `Overlay`? I think that's confusing to read and comprehend why one `Overlay` is handled in a completely different way than all other `Overlay`s. And I'm not sure it's easy to reach the explanation by merely reading the source code (which again prompts me to say "Bravo! Thank you very much!" that you've identified the conflict in the abstractions). I'd really like to have one name per abstraction like our coding style mentions.

2. The source code in `.web` is a duplicate of the source code in `.native` which is forbidden by our coding style. Coding style aside, cannot the `mapStateToProps` addition and the `_callOverlayVisible` check be moved inside 1 file. Maybe in `CallOverlay` itself or `CallOverlayContainer` (I didn't really spend any time of making up a name here, my real target with the inappropriate naming is the analogy with `OverlayContainer`)?

Reply:
Again, great idea! I'll work on sharing that code and finding a new name for it.